realm/realm-library/src/main/java/io/realm/internal/OsSchemaInfo.java

```diff
@@ -16,8 +16,6 @@
 
 package io.realm.internal;
 
-import java.util.List;
```

Comment:
`io.realm.internal.Collection` will have a name conflict in this file, since they are in the same package.